src/app/checkout/accordion.controller.js

```diff
@@ -70,6 +70,7 @@ angular.module('checkoutModule')
         // Discounts and Giftcards
         $scope.discounts = {
             apply: applyDiscount,
+            neglect: discountNeglect,
```

Comment:
Let's change **neglect** to **remove** in every instance in this Pull Request.  In US English, the common definition of **neglect** is similar to _ignore_.  It makes more sense to US based resources to say **remove**.